unit/testing-utils/require_vectors_equal_unordered.h

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************\
+
+ Module: Unit test utilities
+
+ Author: Diffblue Limited.
+
+\*******************************************************************/
+
+#ifndef CPROVER_TESTING_UTILS_REQUIRE_VECTORS_EQUAL_UNORDERED_H
+#define CPROVER_TESTING_UTILS_REQUIRE_VECTORS_EQUAL_UNORDERED_H
+
+#include <testing-utils/catch.hpp>
+#include <vector>
+
+/// Checks whether two vectors are equal, ignoring ordering
+/// \tparam T: The type of the vector contents
+/// \param actual: The vector to check
+/// \param expected: The vector to check against
+template <class T>
+static void require_vectors_equal_unordered(
+  const std::vector<T> actual,
+  const std::vector<T> expected)
```

Comment:
Remove `static` - it's an instruction for linker that a function has been defined only in this file and nowhere else. While it might work on the first use (used only in one file at the moment), it might potentially break later when there are unresovled externals to an already defined specialization of this template.

Also, could you add `&` to `actual` and `expected`? As it is right now, this function will only work with copyable elements or rvalues. That means if you were testing a vector of unique pointers, it would only work with `std::move`.